src/utils/useTrackTopic.js

```diff
@@ -0,0 +1,35 @@
+import { useEffect } from "react";
```

Comment:
Diese Funktion würde ich auch in `trackingUtils.js` stecken, hängt ja stark zusammen.